### ISSUE TITLE
In consult-buffer is it possible to disable preview?

### ISSUE THREAD

**Issue by dmgerman** [minad/consult#812]
I am using window dedications. That means that the buffer will always be displayed in a given way. The problem is that
consult-buffer, when it displays a preview of the one of the dedicated buffers, switches to that window and continues the previews there, instead of in the window where consult-buffer is called from. This is particularly bad when using tabs (tab-bar-mode).

     (setq display-buffer-alist (list
             '("\\*Help\\*"
               (display-buffer-reuse-window display-buffer-pop-up-window)
               (inhibit-same-window . t))
             '("mu4e"
               (display-buffer-in-tab)
               (tab-name . ":email")
               )
             '("\\*org-agenda\\*"
               (display-buffer-in-tab)
               (tab-name . ":agenda")
               )
             ))
         )

For example, assume that the previous display-buffer alist is created. And we are in tab-bar-mode. We have two tabs, one ":email" (with a window containing the buffer mu4e) and the other the current one, where we start consult-buffer.

When the preview shows 'mu4e' the tab changes to the ":email" tab, and the rest of the previews are shown in that tab.
If one presses C-g, the original tab is displayed but now the tab ":email" has whichever was the last buffer previewed.

Basically, after going over a bunch of previews, all my previously dedicated windows have now completely different buffers.

Supporting window dedications would help, but there is a lot of complexity involved in supporting frames and tabs.

Above all, thank you very much for this package.

**Comment by minad** [comment]
Yes, disabling preview is possible. Please take a closer look at the readme first.
